front/src/components/ItemView.tsx

```diff
@@ -1,7 +1,257 @@
+        ...row,
+        Status: status,
+      }))
+    );
+  };
+
+  const deleteItem = (id: string) => {
+    setRows((prevRows) => prevRows.filter((row) => row.ID !== id));
+  };
+
+  const changeStatus = (id: string) => {
+    setRows((prevRows) =>
+      prevRows.map((row) => {
+        if (row.ID === id) {
+          return {
+            ...row,
+            Status: !row.Status,
+          };
+        }
+        return row;
+      })
+    );
+  };
+  const [open, setOpen] = React.useState(false);
+
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const handleAgree = () => {
+    deleteItem('DEV');
+    handleClose();
+  };
+
+  return (
+    <Box sx={
+      {
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'center',
+        alignItems: 'center',
+        width: '80%',
+        margin: 'auto',
+        padding: '20px',
+      }
+
+    }>
+      <Typography variant='h1' sx={
+        {
+          alignItems: 'center',
+          justifyContent: 'center',
+          color: 'black',
+          fontSize: '4rem',
+          fontWeight: 'bold',
+          textAlign: 'center',
+          marginBottom: '2rem',
+        }
+      }>Item Details</Typography>
+      <BasicSelect onChange={handleImportantChange}
+        sx={{
+          display: 'flex',
+          marginBottom: '1rem',
+          width: '15%',
+
+        }}
+      />
+      <TableContainer component={Paper} sx={{
+        borderRadius: '10px',
+      }}>
+        <Table sx={{ minWidth: 700, }} aria-label="customized table">
+          <TableHead>
+            <TableRow>
+              <StyledTableCell>ID</StyledTableCell>
+              <StyledTableCell align="center">Name</StyledTableCell>
+              <StyledTableCell align="center">Age</StyledTableCell>
+              <StyledTableCell align="center">Level</StyledTableCell>
+              <StyledTableCell align="center">Message</StyledTableCell>
+              <StyledTableCell align="center">Important</StyledTableCell>
+              <StyledTableCell align="center">Status</StyledTableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {rows.map((row) => (
+              <StyledTableRow key={row.ID}>
+                <StyledTableCell component="th" scope="row">
+                  {row.ID}
+                </StyledTableCell>
+                <StyledTableCell align="center">{row.Name}</StyledTableCell>
+                <StyledTableCell align="center">{row.Age}</StyledTableCell>
+                <StyledTableCell align="center">{row.Level}</StyledTableCell>
+                <StyledTableCell align="center">{row.Message}</StyledTableCell>
+                <StyledTableCell align="center">
+                  {row.Important ? 'True' : 'False'}
+                </StyledTableCell>
+                <StyledTableCell align="right">
+                  {row.Status ? 'Open' : 'Closed'}
```

Comment:
- [ ] check for the status to be (`open` OR `close`) not( `true` and `false`) as it not boolean but string

- [ ] also use <>open</> and <>close</> instead of the normal qoutes ''